docs/sources/datasources/google-cloud-monitoring/_index.md

```diff
@@ -1,5 +1,5 @@
 +++
-title = "Cloud Monitoring"
+title = "Google Cloud Monitoring"
 description = "Guide for using Google Cloud Monitoring in Grafana"
 keywords = ["grafana", "stackdriver", "google", "guide", "cloud", "monitoring"]
 aliases = ["/docs/grafana/latest/features/datasources/stackdriver", "/docs/grafana/latest/features/datasources/cloudmonitoring/"]
```

Comment:
@achatterjee-grafana - Please add the old path to the aliases list.

Reply:
Added.